tap_zoho_inventory/auth.py

```diff
@@ -52,5 +52,5 @@ def create_for_stream(cls, stream) -> ZohoInventoryAuthenticator:  # noqa: ANN00
         return cls(
             stream=stream,
             auth_endpoint="https://accounts.zoho.com/oauth/v2/token",
-            oauth_scopes="ZohoInventory.items.READ,ZohoInventory.salesorders.READ,ZohoInventory.contacts.READ,ZohoInventory.purchaseorders.READ",
+            oauth_scopes="ZohoInventory.fullaccess.all",
```

Comment:
I don't think it's smart to request such a broad scope, probably better to be specific in what scopes you need

Reply:
Fixed